### ISSUE TITLE
When querying from the UI - dont send all the attributes.

### ISSUE THREAD

**Issue by Astn** [Astn/ekati#96]
The UI would be good to have attributes on the first 10 or so nodes, but after that the UI should request attributes for specific nodes they want to show the user.

For the case when all attributes are wanted for export or something, then we will have a full mode that returns all the attributes, or lets you specify which keys you want returned.

**Comment by Astn** [comment]
Thinking that some syntax like this could be used to control property filters:
```
get "*" 
        |> include ^:^ , *:^ , ^"foo":^, ^"foo":*, *:^"foo"
        |> exclude "age":* , "bio":*
	    |> follow * 1
	    |> limit 500
```

**Comment by Astn** [comment]
fixed in #102
